BGL/include/CGAL/boost/graph/internal/Has_member_clear.h

```diff
@@ -0,0 +1,26 @@
+#ifndef CGAL_HAS_MEMBER_CLEAR_H
+#define CGAL_HAS_MEMBER_CLEAR_H
```

Comment:
@sloriot @bo0ts: License issue. That header should have the LGPLv3+ license notice.